bin/all_sky_search/pycbc_fit_sngls_by_template

```diff
@@ -212,7 +212,10 @@ if args.gating_veto_windows:
         raise ValueError("Gating veto window values must be negative before "
                          "gates and positive after gates.")
     if not (gveto_before == 0 and gveto_after == 0):
-        gate_times = np.unique(trigf[args.ifo + '/gating/auto/time'][:])
+        autogate_times = np.unique(trigf[args.ifo + '/gating/auto/time'][:])
+        if args.ifo+'/gating/file' in trigf:
+            detgate_times = trigf[args.ifo + '/gating/file/time'][:]
+            gate_times = np.concatenate((autogate_times, detgate_times))
         gveto_segs = events.veto.start_end_to_segments(gate_times + gveto_before,
```

Comment:
this is not going to work properly in the case of not having any detechar gates (code won't enter into the if loop and it won't know what `gate_times` is)
I think it's ok

```
autogate_times = np.unique(trigf[args.ifo + '/gating/auto/time'][:])
    if args.ifo + '/gating/file' in trigf:
        detgate_times = trigf[args.ifo + '/gating/file/time'][:]
    else:
        detgate_times = []   
gate_times = np.concatenate((autogate_times, detgate_times))